author/iso/topics/markup.adoc

```diff
@@ -100,6 +100,9 @@ The entries in the Bibliography are reordered (and, for numerical references, re
 
 The bracketed reference numbers are expected to be correct and in order (accounting for the fact that references to standards will end up numbered): they are not overriden in rendering.
 
+[[note_general_doc_ref_bib]]
+NOTE: This subsection supplements link:https://www.metanorma.com/author/topics/document-format/bibliography[References & Bibliography] in general Metanorma documentation. 
```

Comment:
In link URL, instead of specifying protocol and domain name, it’s better to use relative paths, like this:

```
link:/author/topics/document-format/bibliography[References & Bibliography]
```